tools/maven/scalastyle-config.xml

```diff
@@ -68,11 +68,11 @@
    <parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter>
   </parameters>
  </check>
- <check level="error" class="org.scalastyle.scalariform.ObjectNamesChecker" enabled="true">
```

Comment:
Can we use a custom scalastyle for `flink-table` instead of modifying the global one?